presto-main/src/main/java/com/facebook/presto/sql/planner/SymbolToInputRewriter.java

```diff
@@ -40,8 +44,22 @@ public SymbolToInputRewriter(Map<Symbol, Integer> symbolToChannelMapping)
     public Expression rewriteQualifiedNameReference(QualifiedNameReference node, Void context, ExpressionTreeRewriter<Void> treeRewriter)
     {
         Integer channel = symbolToChannelMapping.get(Symbol.fromQualifiedName(node.getName()));
-        Preconditions.checkArgument(channel != null, "Cannot resolve symbol %s", node.getName());
```

Comment:
Remove in efebbd90a82739fe067f2d86a92b1bde998086fd and will add back in a later commit.